platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,515 @@
+
+Standard processes for creating and evolving new capabilities exist, enabling anyone in the organization to contribute a solution that meets expectations. Continuous delivery processes for platform capabilities and features enable regular rollout and rollback. Large changes are planned and coordinated as they would be for customer-facing product changes.
+
+##### Characteristics:
+
+* Application teams request services from platform teams first before creating them.
+* New services must adhere to standard practices such as standard interfaces, documentation, and governance.
+* Upgrade processes are documented and consistent across versions and services.
+* Where the capability provider does not manage an upgrade, they provide tooling and support to the users for minimal impact.
+
+##### Example Scenarios:
+
+* The organization is going to upgrade to RHEL 9. In doing so, each application team needs to validate that their software continues to work. In order to enable this testing phase the centralized compute team is setting up test environments for each team with the correct software and OS versions.
+
+#### Level 4, Optimizing - Managed services
+
+The lifecycle of each capability is managed in a standardized, automated way. Capabilities, features and updates are delivered continuously with no impact on users. Any large changes instigated by platform providers include migration plans for existing users with defined responsibilities and timelines.
+
+Platform capability providers take on the brunt of responsibility for maintenance, but there is a clear contract — a "shared responsibility model" — describing the responsibilities of users, enabling both sides to operate mostly autonomously.
+
+##### Characteristics:
+
+* A shared ownership model clearly defines who is responsible for platforms and their capabilities and what is expected of users. 
+* Teams script both the execution of the upgrade and any rollback strategies to keep risk and impact low.
+
+##### Example Scenarios:
+
+* The users of virtual machines are not required to manage anything to do with version upgrades. Their only requirement is to have a stage in their delivery pipeline that contains a representative smoke test. They are then asked to declare their application as having lower risk tolerance so as to wait for a fully hardened upgrade or higher tolerance to become an early adopter. The virtual machine capability then manages the automated release of upgrades including rollbacks after either smoke test or canary release failures.
+
+{{< /tab >}}
+{{< tab tabName="Measurement">}}
```

Comment:
For some reason, this is rendering the operations content

|-|-|
|-|-|
|<img width="968" alt="image" src="https://github.com/cncf/tag-app-delivery/assets/5528307/b789a842-7e5a-451e-8c11-ba3cab4912e9">|<img width="928" alt="image" src="https://github.com/cncf/tag-app-delivery/assets/5528307/e6b4123b-89af-469e-9093-2a73711e57f7">|